lib/clusterable/commitment.rb

```diff
@@ -4,19 +4,50 @@
 
 module Clusterable
 
-  # A commitment
+  # A shared print commitment
   class Commitment
     include Mongoid::Document
-    field :ocn, type: Integer # , type: OCLCNumber
+    field :uuid, type: String
     field :organization, type: String
+    field :ocn, type: Integer
+    field :local_id, type: String
+    field :oclc_sym, type: String
+    field :committed_date, type: DateTime
+    field :retention_date, type: DateTime
+    field :local_bib_id, type: String
+    field :local_item_id, type: String
+    field :local_item_location, type: String
+    field :local_shelving_type, type: String
```

Comment:
This could be restricted to one of a small set of strings, if we care to.

Reply:
"this" = local_shelving_type, which in the (human) spec doc is restricted to one of ['cloa','clca','sfca','sfcahm','sfcaasrs'].